### PR TITLE
CA-753 Added participant order function with test

### PR DESCRIPTION
**Pull request by Galzk**
Still need to test on the app **and add sort by join time** (done ✅)

### CODE REVIEW

**Comment by Galzk** [comment]
We should test it before deployment

File: src/components/ParticipantStrip/ParticipantStripCollaboration/ParticipantStripCollaboration.tsx [hunk 1]
```diff
@@ -25,9 +26,10 @@ export default function ParticipantStripCollaboration() {
   const {
     room: { localParticipant },
   } = useVideoContext();
-  const participants = useParticipants();
+  const unorderedParticipants = useParticipants();
```
Comment (Galzk) [inline src/components/ParticipantStrip/ParticipantStripCollaboration/ParticipantStripCollaboration.tsx]: Under @talr direction:
Create new hook that's called useSortedParticipants and move this functionality there

**Review by Galzk — COMMENTED** [review]
Tested and ready to be merged!

File: src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import EventEmitter from 'events';
 import React from 'react';
-import ParticipantStripGrid, { ParticipantStripGridProps } from './ParticipantStripGrid';
+import ParticipantStripGrid from './ParticipantStripGrid';
```
Comment (talr) [inline src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.test.tsx]:
```suggestion
import ParticipantGrid from './ParticipantGrid';

```

File: src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.tsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ import useParticipants from '../../../hooks/useParticipants/useParticipants';
 import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
 import useSelectedParticipant from '../../VideoProvider/useSelectedParticipant/useSelectedParticipant';
 import Participant from '../../Participant/Participant';
+import sortParticipants from '../../../utils/sortParticipants';
```
Comment (talr) [inline src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.tsx]: Remove line

File: src/state/index.tsx [hunk 4]
```diff
@@ -77,15 +77,14 @@ export default function AppStateProvider(props: React.PropsWithChildren<{}>) {
         process.env ? 'process.env: ' + JSON.stringify(process.env) : 'not yet initialised'
       }`
     );
+    const PUBLIC_URL = process.env.PUBLIC_URL || process.env.REACT_APP_PUBLIC_URL;
     console.log(
```
Comment (talr) [inline src/state/index.tsx]: Can we replace all console.log with eScribers logger? Which also write on the console and give us better observability.

File: src/state/index.tsx [hunk 5]
```diff
@@ -77,15 +77,14 @@ export default function AppStateProvider(props: React.PropsWithChildren<{}>) {
...
-          : 'not yet initialised'
+        PUBLIC_URL ? 'process.env.PUBLIC_URL: ' + JSON.stringify(PUBLIC_URL) : 'not yet initialised'
       }`
     );
 
-    await fetch(`${process.env.PUBLIC_URL}/config.json`)
+    await fetch(`${PUBLIC_URL}/config.json`)
       .then(
         response => {
           console.log('response from fetch received');
```
Comment (talr) [inline src/state/index.tsx]: Same

**Review by talr — CHANGES_REQUESTED** [review]

File: src/state/index.tsx (hunk 4 above)
Comment (Galzk) [reply src/state/index.tsx]: Out of scope, we don't have a logger initialized in this project

**Review by talr — APPROVED** [review]

**Pull request merged**